brain_games/scripts/brain_calc.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+from brain_games.calc import calc
+from brain_games.cli import welcome_user
+from brain_games.game import game
+
+
+def main():
+    name = welcome_user()
```

Comment:
both scripts run this function at the start, maybe move it into `game` function?